includes/excerpts.php

```diff
@@ -16,15 +16,20 @@ function today_get_excerpt( $post, $length=TODAY_DEFAULT_EXCERPT_LENGTH ) {
 	if ( ! ( $post instanceof WP_Post ) ) return '';
 
 	$excerpt = '';
-	$custom_filter = function( $l ) use ( $length ) {
-		return $length;
-	};
 
 	setup_postdata( $post );
```

Comment:
The `setup_postdata()` call could be moved to the top of the `else` statement, since it only needs to be called if we're calling `get_the_excerpt()`.